CHANGELOG.md

```diff
@@ -11,6 +11,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 - `PromptTask.conversation_memory` for setting the Conversation Memory on a Prompt Task.
 - `Structure.conversation_memory_strategy` for setting whether Conversation Memory Runs should be created on a per-Structure or per-Task basis. Default is `Structure.ConversationMemoryStrategy.PER_STRUCTURE`.
+- `BaseFileLoader.save()` method for saving an Artifact to a destination.
```

Comment:
Wanna add an example in the docs too? Maybe in the same docs for using the load method?

Reply:
Added example